lib/utils/import/index.js

```diff
@@ -0,0 +1,49 @@
+import { isEmpty, get, pick } from 'lodash';
+
+const propertyWhitelist = [
+  'content',
+  'creationDate',
+  'lastModified',
+  'modificationDate',
+  'pinned',
+  'tags',
+  'deleted',
+];
+
+const importNotes = (notes = {}, noteBucket) => {
+  if (isEmpty(notes)) {
+    throw new Error('No notes to import.');
+  }
+
+  if (!get(notes, 'activeNotes') || !get(notes, 'trashedNotes')) {
```

Comment:
```suggestion
  if (!notes.activeNotes || !notes.trashedNotes) {
```

I’m wondering if this would be more readable and functionally equivalent in this case.

Reply:
Also this should be an `&&` not an `||` 😬